### ISSUE TITLE
How to define a loss function that needs to input numpy array(not tensor) when build a tensorflow graph?

### ISSUE THREAD

**Issue by phoenix1992** [tensorflow/tensorflow#24479]
hello,
I want to add a constraint option in my loss function. The definition of this constraint option needs numpy array type as input. So, I can not define it as a tensor type as a graph node in tensorflow. How can I define this part in graph so as to join in the network optimization?
